### DISCUSSION TITLE
Client migration guidance

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [IDI-Systems/acre2#1296] [Q&A]
Saw the [drop](https://github.com/IDI-Systems/acre2/releases/tag/v2.12.0.1056) today, good stuff, thank you. Any suggestions re: tracking with the latest client? `v3.6.2` is available, currently running `v3.6.0`. Thanks...

**Comment by jonpas** [comment]
Can you expand on what the question is? I don't fully understand.

**Reply by mwpowellhtx** [reply]
Is it safe to migrate to `v3.6.2`? Any ACRE2 side effects doing so?

**Reply by jonpas** [reply]
No issues have been reported so far. It is almost identical to v3.6.0 with some fixes. It is considered safe from our perspective.

**Comment by mwpowellhtx** [comment]
Thanks 👍
